### DISCUSSION TITLE
How do set string as a variable?

### DISCUSSION THREAD

**Discussion by VillianousJ0ker** [HaxeFlixel/flixel#3018] [Q&A] [answered]
Currently I have been working on a small little game with friends as a small submission for a project, and right now because of how new I am to Haxe and that stuff, is it possible to set a FlxText string or something to a variable always? Or atleast have some kind of big score text at the top of the screen?

**Comment by moxie-coder** [comment, marked as answer]
you could do `exampleFlxText.text = string;`

but the discussions and discord exist

**Comment by VillianousJ0ker** [comment]
oh sry mb, thx for the help tho!

**Comment by moxie-coder** [comment]
> oh sry mb, thx for the help tho!

np
